spec/models/organisation_spec.rb

```diff
@@ -213,7 +213,13 @@
     let(:ip4) { create(:ip, location: location2) }
     let(:ip5) { create(:ip, location: location3) }
 
-    before { ip1; ip2; ip3; ip4; ip5 }
+    before do
+      ip1
+      ip2
+      ip3
+      ip4
+      ip5
```

Comment:
You can save yourself the trouble by immediately invoking the `let` blocks with the special [`let!` form](https://relishapp.com/rspec/rspec-core/v/3-9/docs/helper-methods/let-and-let).